index.html

```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
         <li><a href="#" title="Angellist"><img src="img/angellist.svg" alt="angellist logo"></a></li>
       </ul>
     </section>
+     <!-- Work -->
+    <section id="work">
+      <div class="work-first flex-column">
+        <div class="work-description flex-column">
+          <div> <img src="img/Portfolio-1.png" alt=""></div>
+          <div>
+            <div class="specification">
+              <h2>Hossain</h2>
+              <div class="flex">
+                <h4>CANOPY</h4>
+                <!-- <p>&#x2022;</p>
+                <P> Full Stack Dev</P>
+                <p>&#x2022;</p>
+                <p>2023</p> -->
+                <ul class="flex">
+                  <li>Full Stack Dev</li>
+                  <li>2023</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-description flex">
+          <button class="language-btn"> <p>html</p></button>
+          <button class="language-btn"><p>css</p></button>
+          <button class="language-btn"><p>Javascript</p></button>
+        </div>
```

Comment:
- [ ] These list of technologies are meant to be inside a `ul` tag and them being a `li` item, I suggest you update these. :+1: 
- [ ] Also looking at the given Figma design, your text font tends to be smaller than the given design, also your design has a border that is not in the given design, I suggest you update this accordingly. :+1: :100: 

- - Your Current Design:
![Screenshot from 2023-06-07 09-16-40](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/107632780/89f13c1b-1bb3-457a-8eee-34c2e5797cc5)

- - Figma design:
![Screenshot from 2023-06-07 09-16-27](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/107632780/16a9ef30-ac67-41c4-a664-fb7f66530436)

- [ ] I also suggest you take a look at the margin top and bottom, the given design tends to have more margin than your current implementation. :+1: :smile: 

**Remember to implement this change in all the technology lists in each project card**